core/Names.cc

```diff
@@ -291,6 +293,14 @@ NameRef NameRef::prepend(GlobalState &gs, string_view s) const {
     return gs.enterNameUTF8(nameEq);
 }
 
+NameRef NameRef::lookupMangledPackageName(const GlobalState &gs) const {
```

Comment:
This isn't used anywhere else, so I'd be happy to move it inline somewhere. It just felt out-of-place to have string manipulation inline, and this might be robust to future internal package name changes. (It can't be used to generate the package name since it needs a `const GlobalState&`, so the `Name` needs to already exist, but we can replicate the string-munging into an `enterMangledPackageName` and move that logic out of the packager if we want them both to live in the same place.)